src/utils/Hair.js

```diff
@@ -191,6 +191,100 @@ class HairStrand {
       array.push(z);
     }
   }
+
+  /**
+   * There will be three styles for verlet particle spreads
+   * 1 - each hair segment is the same length
+   * 2 - each new hair segment cuts the base and the following vertex in half
+   * 3 - spread factor of 2/3 with the base to 1 and 1 to 2 segments being equal length
+   */
+  generateInitialVerletParticles(normal_x, normal_y, normal_z, base_x, base_y, base_z, spread_type){
+    //////ADD BASE AND TAIL VERLET PARTICLES TO ARRAY//////
+    let dampen_factor = 0.96 + this.getRandomWiggle(0.06);
+    this.verlet_parts.push(new VerletParticle(base_x, base_y, base_z, true, dampen_factor, this.min_pearl_rad)); //BASE
+
+    let temp_x = normal_x * this.length + base_x + this.getRandomWiggle(0.02);
+    let temp_y = normal_y * this.length + base_y + this.getRandomWiggle(0.02);
+    let temp_z = normal_z * this.length + base_z + this.getRandomWiggle(0.02);
+    dampen_factor = 0.96 + this.getRandomWiggle(0.06);
+    let pearl_radius = this.max_pearl_rad;
+    pearl_radius += this.getRandomWiggle(0.02);
+    this.verlet_parts.push(new VerletParticle(temp_x, temp_y, temp_z, false, dampen_factor, pearl_radius)); //TAIL
+    ////////////
+
+    switch(spread_type){
```

Comment:
Could be worth doing a SpreadType "enum" (enums aren't actually a thing in JS) to increase readability.

```javascript
export const SpreadType = Object.freeze({
  EVEN: 1,
  LOGARITHMIC: 2,
  OTHER: 3, // or however you would describe this one
});
```

And then you could use that in the following fashion:
```javascript
import HairStrand, { SpreadType } from 'utils/Hair';

const strand = new HairStrand({ /* ... */, s_type: SpreadType.LOGARITHMIC });
```

Reply:
Yeah, that's a good idea. Just provides more clarification for what's going on.